azure-pipelines-gc.yml

```diff
@@ -0,0 +1,27 @@
+resources:
+  containers:
+    - container: ubuntu_x64_build_container
+      image: microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-c103199-20180628134544
+
+# Trigger builds for PR's targeting master
+pr:
+  branches:
+    include:
+    - master
+  paths:
```

Comment:
Should probably add some include paths here for just the GC area

Reply:
Done! Would this be enough? I'm setting it to trigger anytime the main script, or any source files are modified.